demo/launch/ik_clearance_cost.launch.py

```diff
@@ -1,55 +1,27 @@
 import os
```

Comment:
```suggestion

```

Reply:
Also, for what it's worth, I couldn't get this example to work when testing, but I don't see that being related to this PR at all.

Reply:
Do you mind sharing the error you're getting?

Reply:
I think the issue is actually a problem with rosparam_shortcuts that looks related to your PR https://github.com/PickNikRobotics/rosparam_shortcuts/pull/27

Here's the error:

```
[ik_clearance_cost-1] terminate called after throwing an instance of 'rclcpp::exceptions::UninitializedStaticallyTypedParameterException'
[ik_clearance_cost-1]   what():  Statically typed parameter 'cumulative' must be initialized.
[ERROR] [ik_clearance_cost-1]: process has died [pid 642360, exit code -6, cmd '/workspaces/ros2-rolling-ws/install/moveit_task_constructor_demo/lib/moveit_task_constructor_demo/ik_clearance_cost --ros-args --params-file /tmp/launch_params_etkxjhf0 --params-file /tmp/launch_params_c324op82 --params-file /tmp/launch_params_26xnvx2h'].
```
I also noticed a small typo, please look at #366 if you get a chance!